src/state.js

```diff
@@ -71,10 +67,10 @@ export const getInitialState = (): State => ({
     lastFetchingError: null,
     utxos: null,
   },
-  trans: l10n.translations,
   isOnline: true, // we are online by default
   isAppInitialized: false,
   isKeyboardOpen: false,
+  tos: 'Terms Of Use',
```

Comment:
I assume this is a placeholder value until the ToS file is loaded up for a language an inserted into this variable?

Do we really want the entire text of the ToS stored in the state? Feels like the file should just be opened on-demand based on the language selected like is done in Yoroi-Frontend

Reply:
I'll take care of this later on, but now I think it'd be better to proceed with the merge.

Reply:
@v-almonacid can you create an issue in clubhouse to keep track of this?